backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/impl/HUQtyService.java

```diff
@@ -115,7 +115,7 @@ public HuForInventoryLine toHuForInventoryLine(
 				.quantityBooked(huQty)
 				.quantityCount(newQty)
 				.productId(huProductStorage.getProductId())
-				.storageAttributesKey(handlingUnitsBL.getStorageRelevantAttributesKey(hu))
+				.storageAttributesKey(handlingUnitsBL.getAttributesKey(hu))
```

Comment:
Q: wondering if it would make sense to rename `storageAttributesKey` to `attributesKey`. wdyt?